cytotable/presets.py

```diff
@@ -153,6 +153,59 @@
                 AND per_nuclei.Metadata_Nuclei_Number_Object_Number = per_cytoplasm.Metadata_Cytoplasm_Parent_Nuclei
             """,
     },
+    "cellprofiler_legacy": {
+        # version specifications using related references
+        "CONFIG_SOURCE_VERSION": {
+            "cellprofiler": "v2.X",
```

Comment:
Looking at the nam of this preset and the version here, would it make sense to specify a version range for CellProfiler? For example, `v2.x-v3.x`, or something like this?

Reply:
> When did the renaming of the column started to happen? 

I do not know

Reply:
I'm also not sure about the name changes per versions. I brought this up just in case you knew specific versions this preset would apply to beyond just 2.x. If this is uncertain, no changes necessary 🙂.

Reply:
Left as-is given uncertainty.